### PR TITLE
Update Cohort Duration

### PR DESCRIPTION
**Pull request by Bamieh**
Update Cohort Duration to 10 weeks (2 and a half months) instead of 6 months.

@nodejs/mentorship According to last meeting, everyone agreed that 2-3 months duration is good. I've used **10 weeks** as it is round number and comes in the middle of that range!

### CODE REVIEW

**Review by detrohutt — APPROVED** [review]
I think we still need to discuss how or if we will perform kickoff meetings moving forwards, but this all looks good in the meantime!

**Review by dshaw — APPROVED** [review]
Let's align and land in today's (2019-01-25) meeting.

File: README.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ always being or finding the solution themselves.
 
 # The Mentorship Program Structure
 
-The mentorship duration will be 6 months, during which the mentor will provide
+The mentorship duration will be 10 weeks, during which the mentor will provide
```
Comment (dshaw) [inline README.md]:
```suggestion
The mentorship duration will be 10 weeks (2.5 months), during which the mentor will provide
```

**Review by dshaw — CHANGES_REQUESTED** [review]
Please change 10 weeks to 2.5 months.

**Pull request merged**